k6/src/scenarios/test/eth_call.js

```diff
@@ -35,6 +35,8 @@ const {options, run} = new TestScenarioBuilder()
     )
   )
   .check(methodName, (r) => isNonErrorResponse(r))
+  .testDuration("3s")
```

Comment:
Q: why were these needed?

Reply:
Since `eth_call` is resolved using `mirror_node` response time can vary greatly and in order for the tests to pass, I custom override the default time to respond wich is 1 second (or defined by env property `DEFAULT_MAX_DURATION` to 3 seconds.